docs/farming-&-staking/staking/staking.md

```diff
@@ -127,57 +77,52 @@ Alternatively, you can use [Subscan](https://subspace.subscan.io/) which is a li
 
+:::note
 - If an operator is initiating a withdrawal, then their remaining balance should be at least the minimum operator stake, otherwise the request is rejected.
 - If a nominator is initiating a withdrawal, and the remaining balance is less than the operator-defined minimum nominator stake, then the total nominator stake is unlocked otherwise, only the requested amount is unlocked.
+:::
 
-1. Proceed to [PolkadotJS](https://polkadot.js.org/apps/#/explorer).
-2. Navigate to Developer -> Extrinsics. 
-3. Select the account you want to use in `using the selected account`.
-4. Select `domains` under `submit the following extrinsic` and choose `withdrawStake(operatorId, withdraw)` 5n the dropdown.
-6. Choose an operator by selecting an `operatorId` - in the example, it's set to `17`.
-7. Choose the withdrawal amount in the `withdraw` field - you can specify to withdraw `all` or `some` staking amount. 
+1. Proceed to the staking tab on [Astral](https://explorer.subspace.network/gemini-3h/staking) and connect your wallet.
 
-:::note
-Example of withdrawal of 1 TSSC stake amount from nominating an operator `17`.
-:::
+![NStaking-1](/img/doc-imgs/operators-staking/NStaking-1.png)
+
+2. Select the wallet you would like to connect. Both **Subwallet** and **PolkadotJS** wallets are supported.
 
-![Staking-23](/img/doc-imgs/operators-staking/Staking-23.png)
+![NStaking-2](/img/doc-imgs/operators-staking/NStaking-2.png)
+
+3. Enter your password to give an access to your wallet.
+
+![NStaking-3](/img/doc-imgs/operators-staking/NStaking-3.png)
+
+4. Select the account you'd like to use form the dropdown menu. You can click the button in the upper left corner to see available token balances for the selected account. 
+
+![NStaking-4](/img/doc-imgs/operators-staking/NStaking-4.png)
+
+![NStaking-5](/img/doc-imgs/operators-staking/NStaking-5.png)
+
+6. Once you picked the operator to withdraw your stake from, click on **Actions** -> **Withdraw**.
+
+![N-nominators-6](/img/doc-imgs/operators-staking/N-Nominators-6.png)
+
+7. You will get a pop-up requesting how much you want to withdraw.  Use the slider to determine what percentage you want to withdraw or you can click the max button to withdraw your entire stake.
+
+![N-nominators-7](/img/doc-imgs/operators-staking/N-Nominators-7.png)
 
 
 Once the withdrawal is submitted, it's finalized after the **domain epoch** is completed. All the withdrawn funds are unlocked after the **locking period** is complete. The current **locking period** is set to 28,800 blocks, or about 48 hours. The locking period is necessary to ensure that the domain block executing the withdrawal is confirmed and not challenged by a fraud proof and to increase the economic stability of domains.
 
-After the **locking period**, the withdrawn amount can be unlocked in the user's account with the `unlock_funds` extrinsic.
+After the **locking period**, the withdrawn amount can be unlocked in the user's account.
```

Comment:
Information in the paragraph above is outdated, the locking period is 14,400 blocks, or about 24 hours.